### ISSUE TITLE
Onboarding design

### ISSUE THREAD

**Issue by franzaps** [zapstore/zapstore#180]
Given that app developers might recommend Zapstore to download their app -- how do we make the experience as seamless as possible? Also consider people who don't have zs installed

Add visual steps and/or documentation for hand holding

**Comment by franzaps** [comment]
@alltheseas no rush on this one but for a later design call

**Comment by franzaps** [comment]
https://njump.me/note134ekjt2yp3gucrl4d8fxq3wljlmh2wg7vntsqh542k8hc662wv4qx05qzq

**Comment by alltheseas** [comment]
> Given that app developers might recommend Zapstore to download their app -- how do we make the experience as seamless as possible? Also consider people who don't have zs installed
> 
> Add visual steps and/or documentation for hand holding

Yes, and app developers like @vitorpamplona may choose to recommend other apps (e.g. ntfy for push notifications) to be downloaded directly via Zapstore. Current/status quo behavior: link to app website; link to google play store.

**Comment by vitorpamplona** [comment]
I need to recommend installing Amber to login, and Citrine as the local backup. Then for notification, users need to choose between Google play services (no install), installing ntfy or Pokey. Then I need them to install a wallet. It could be a Minibits cashu with nip60 on Amethyst or coinos/strike. 

It all cases, Amethyst needs to either let the user choose in our screen OR know which option the user has chosen. We can see which apps are installed, so we will need another way.

**Comment by alltheseas** [comment]
How might we reduce the complexity inherent in all the choices, and installations @vitorpamplona wants to surface to and empower the Amethyst user with above @franzaps ?

"One click install many apps (grandma mode)"? 

### realted

[install apps recommended by my friend](https://github.com/zapstore/zapstore/issues/49)

**Comment by vitorpamplona** [comment]
Maybe each app has a "best companions" record that the app can just call?

**Comment by alltheseas** [comment]
>that the app can just call?

By "the app" do you mean Amethyst, Zapstore, or both?

**Comment by vitorpamplona** [comment]
Each app in the store can have a list of companion apps. Or maybe a few lists for different user choices.

**Comment by alltheseas** [comment]
example comprehensive user story of personalized invite link for social/tweet app: https://github.com/damus-io/damus/issues/1696

This relates to app backup list/restore apps from backup list
#20
